packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -19,7 +19,7 @@
         </span>
         <todo-label v-for="label in todo.labels" :todoLabel="label" :key="label.name ? label.name : label" />
       </div>
-      <todo-text v-if="isEditing" v-on:cancel="cancel" v-on:change="updateTitle" v-bind.sync="{ initialTodoTitle: completeText }" :key="`updateTodo-${todo.todoId}`" />
+      <todo-text v-if="isEditing" v-on:cancel="cancel" v-on:change="updateTitle" v-bind.sync="{ initialTodoTitle: completeText }" :key="`updateTodo-${todo.todoId}`" :parentId="todo.parentId" />
```

Comment:
does the todo-text need the parentId? I guess, it's only needed in `updateTitle` below, right?

Reply:
If we don't have this, the `updateTitle` will be called without a `parentId`. I'm not sure why, but along the way, it will set the `parentId` to `null`. It may be done by mongoose again to set a default here if not undefined?

Ahh and now I understand what you meant. True, that has to be the case. I'll remove it here.

Reply:
Removed it here, kept it below in the `updateTitle` function